docs/aziende-di-consulenza-vs-prodotto/README.md

```diff
@@ -0,0 +1,99 @@
+In altri casi, questa sensazione di distacco è meno forte. In alcune aziende di consulenza infatti si lavora in modalità "_progetti chiavi in mano_". Con questa modalità, i progetti dei clienti vengono presi in carico dall'azienda di consulenza, curati e sviluppati interamente dai propri team. Questo permette a chi sviluppa di avere un team di riferimento più stabile rispetto a quando si lavora presso il cliente, aumentando il senso di appartenenza.
+
+### Possibilità di carriera
+
+Le aziende di consulenza puntano come obiettivo principale ad acquisire commesse dalle aziende clienti e puntare alla loro soddisfazione.
+
+Se questo obiettivo è perseguito in maniera sana, per un/a developer lavorare in azienda di consulenza significa potenzialmente lavorare con diverse tecnologie e avere la possibilità di formarsi in modo continuo e variegato.
+Questo si potrebbe tradurre in una rapida crescita e una più ampia possibilità di carriera. E spesso, le aziende di consulenza sono il punto di ingresso principale anche per le figure più Junior.
+
+Se questo obiettivo invece è perseguito dall'azienda con disorganizzazione, acquisendo progetti solo con lo scopo di fatturare senza tenere conto della reale disponibilità di un adeguato numero di personale preparato, questo potrebbe influire sul team di sviluppo in forma di mole di lavoro eccessiva, causando demotivazione e frustrazione, e in particolare sulle figure Junior che potrebbero essere abbandonate a loro stesse senza un'adeguata formazione.
+
+Nelle aziende di prodotto invece la responsabilità principale di ogni reparto dell'azienda sarà quello di mantenere il prodotto attivo e attrattivo sul mercato. Questo vale sia per il reparto di marketing e comunicazione, quanto per il reparto tecnico sia a livello di innovazione e sviluppo dello stesso.
+
+Se l'azienda di prodotto ha una direzione precisa e organizzata, per chi sviluppa e ci lavora sarà più naturale puntare alla propria crescita formativa, alla qualità del proprio operato e alla collaborazione e crescita in team.
+
+Per quanto la qualità del prodotto sviluppato è obiettivamente una priorità delle aziende di prodotto, tirando le somme, le possibilità di carriera e di crescita sono comunque fattibili in entrambe le tipologie di azienda, il tutto si riduce a quanto queste siano organizzate e attente alla formazione dei propri dipendenti.
+
+Per una persona Junior o senza una particolare esperienza significativa, è sicuramente più semplice entrare in un'azienda di consulenza poiché nelle aziende di prodotto in genere vengono ricercate persone autonome e con un background solido.
```

Comment:
In questo punto viene data più "certezza" sul fatto che per uno junior sia più semplice entrare, mentre precedentemente viene menzionato, giustamente, come il rischio possa essere di essere parcheggiati dal cliente. Forse lascerei anche qui più spazio di manovra, in fondo in aziende di prodotto "sane" si ha il vantaggio di formarsi in maniere più verticale e con più supporto.

Reply:
@giuppidev chiaramente le differenze e i casi esistono, però credo che le conclusioni finali siano già sufficienti ad ampliare lo spazio di manovra, che ne pensi?

In alternativa, cosa ne pensi se portiamo dentro il capitolo (Siamo già alle 6 approvazioni necessarie) e poi suggeriamo una eventuale ri-visitazione di alcune parti?

Reply:
@Cadienvan si si, probabilmente quel "sicuramente più semplice" mi ha triggerato ma i concetti poi sono abbastanza esplicativi. Vamos dai